src/query/users/src/role_mgr.rs

```diff
@@ -124,21 +123,21 @@ impl UserApiProvider {
             &GrantObject::Global,
             UserPrivilegeSet::available_privileges_on_udf(),
         );
-        let mut public = RoleInfo::new(BUILTIN_ROLE_PUBLIC);
-        public.grants.grant_privileges(
-            &GrantObject::Table(
-                "default".to_string(),
-                "system".to_string(),
-                "one".to_string(),
-            ),
-            UserPrivilegeType::Select.into(),
-        );
+        let public = RoleInfo::new(BUILTIN_ROLE_PUBLIC);
+        // public.grants.grant_privileges(
+        //     &GrantObject::Table(
+        //         "default".to_string(),
+        //         "system".to_string(),
+        //         "one".to_string(),
+        //     ),
+        //     UserPrivilegeType::Select.into(),
+        // );
```

Comment:
if i understand correctly, we've moved the logic about skipping the checks with `system.one` to other place, right?

Reply:
Yes, I think we can 

1. skip these table or dbs privilege check in check function
2. or get the table_id and db_id in there.

Reply:
We need skip table_function, system.one, information_schema

Reply:
> We need skip table_function, system.one, information_schema

Do more check in privilege_access.rs and system.tables/databases/columns